### PR TITLE
Cameras: Convert to es6 class.

### PR DESCRIPTION
**Pull request by linbingquan**

### CODE REVIEW

**Comment by Mugen87** [comment]
This PR breaks `CinematicCamera`.

**Comment by linbingquan** [comment]
> This PR breaks `CinematicCamera`.

Yes, thank you to review, I try to solve it.

File: examples/jsm/cameras/CinematicCamera.js [hunk 1]
```diff
@@ -13,205 +13,205 @@ import {
 import { BokehShader } from "../shaders/BokehShader2.js";
 import { BokehDepthShader } from "../shaders/BokehShader2.js";
 
-var CinematicCamera = function ( fov, aspect, near, far ) {
+class CinematicCamera extends PerspectiveCamera {
```
Comment (Mugen87) [inline examples/jsm/cameras/CinematicCamera.js:16]: This PR is not mergeable as long as you have to modify example code.

Notice that it is not acceptable to just modify `jsm` modules. Until end of this year, `example/js` and `exampes/jsm` have to stay in sync (expect for new files). 

Please revert any changes to `CinematicCamera` and `PerspectiveCamera`.

Comment (linbingquan) [reply examples/jsm/cameras/CinematicCamera.js:16]: How about this https://github.com/mrdoob/three.js/pull/20102

Comment (Mugen87) [reply examples/jsm/cameras/CinematicCamera.js:16]: Closing in favor of #20102.

**Pull request closed without merging**